### DISCUSSION TITLE
Code Submission

### DISCUSSION THREAD

**Discussion by kumkee** [Te-Maramataka/mataripy#10] [Announcements]
Dear teams,
We are entering the code submission phase of the competition. 
Your pushes to your team repositories will all be considered as your submission with the exception of any commits after 11:59 PM on 27 July NZST. 
In your final submission, apart from your codes (and comments/documentation), please also include:

- A text output file logging your Matariki holiday prediction for every 10 years from 1990 to 2990 (100 predictions)
- A YouTube (or your favourite video hosting platform) video link showing how your run your programme for 5 random years generated from [this web link](https://www.random.org/integer-sets/?sets=1&num=5&min=10&max=4000&seqnos=on&commas=on&sort=on&order=index&format=html&rnd=new). In your video, please also include a screenshot of the random generator website with its timestamp.
- In your documentation, please include an instruction on how others can set up the software environment to run your programme on a typical Windows/Linux machine.

**Comment by sttaseen** [comment]
Hi, for the text file, what format would you like the dates to be in, d-M-Y or Y-m-d or d-m-Y?

**Reply by kumkee** [reply]
It doesn't matter, as long as there is no ambiguities.

**Reply by sttaseen** [reply]
ok thank you! By the way, do you use a mobile app? Your replies are lightning fast.

**Reply by kumkee** [reply]
> ok thank you! By the way, do you use a mobile app? Your replies are lightning fast.

GitHub? No. I just got email notification.

**Comment by kumkee** [comment]
Late commit permissions:

- Team Arnav/Jalia: permitted to add apk file
- Team EV: permitted to add video link and text file

Both have been done now. No actual codes have been modified in both team.
